c++/include/orc/ColumnPrinter.hh

```diff
@@ -47,6 +47,6 @@ namespace orc {
   };
 
   ORC_UNIQUE_PTR<ColumnPrinter> createColumnPrinter(std::string&,
-						    const Type& type);
+						    const Type& type, const std::vector<bool>* selectedColumns = nullptr);
```

Comment:
In the public API, you need to use ORC_NULLPTR to compile on non-C++ 0x11.

Please line break your addition to keep it within 80 chars.